### PR TITLE
handle bullet as multi-config on WIN32

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: ea88518a440ced8cae3a051fc41cb19aa5a38e34 available at: https://gist.github.com/5a54e44c5eb49f3c56ee<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5a4a3ce6dd99e054c80c37d0ac7571c9610fc0b0 available at: https://gist.github.com/2fb4264becfcd234abec<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.dmg

**Comment by birarda** [comment]
@AndrewMeadows this definitely isn't ready to go yet, I'll let you know!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9979bbd8b52eb67e73326532e96f4e4ed723090f available at: https://gist.github.com/bdf1b97a4095b8faeeac<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 43da329eb4734777157123c50046aaf4b6df431d available at: https://gist.github.com/71a4b896aa5859f91d18<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: eb646383e99318676000d389ad64cf32a5aea55d available at: https://gist.github.com/5c4abef7380d85beaf04<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.dmg

File: BUILD_WIN.md [hunk 1]
```diff
@@ -16,16 +16,12 @@ If using Visual Studio 2013 and building as a Visual Studio 2013 project you nee
 
 ####nmake & msbuild
```
Comment (murillodigital) [inline BUILD_WIN.md]: nmake alone now

**Comment by hifi-gustavo** [comment]
Build console output for commit: 95c25086d70d06aa71d815c9ecb3d7b5dc727be4 available at: https://gist.github.com/ddf2d36718fafd719401<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e95c197756001edc92084eadf2ba45ecf773775b available at: https://gist.github.com/6ea20f91013280185d74<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4346.dmg

**Comment by PhilipRosedale** [comment]
ready to pull @birarda @AndrewMeadows ?

**Comment by AndrewMeadows** [comment]
@Zappoman says @birarda says this is ready.  So I'll merge.

**Pull request merged**